### PR TITLE
DevTools 106 > Modern web debugging

### PR DESCRIPTION
**Pull request by sofiayem**
Documented:
- https://developer.chrome.com/blog/new-in-devtools-106/#authored
- https://developer.chrome.com/blog/new-in-devtools-106/#async
- https://developer.chrome.com/blog/new-in-devtools-106/#auto-ignore
- https://developer.chrome.com/blog/new-in-devtools-106/#call-stack
- https://developer.chrome.com/blog/new-in-devtools-106/#ignore-nav
- https://developer.chrome.com/blog/new-in-devtools-106/#ignore-search

Publish on Sep 27.

TODO: Document `console.createTask()` in a separate PR.

### CODE REVIEW

File: site/en/docs/devtools/javascript/reference/index.md [hunk 1]
```diff
@@ -275,6 +303,51 @@ inputsAreEmpty (get-started.js:22)
...
+
+Use [the **Page** pane](/docs/devtools/javascript/sources/#files) to navigate the file tree.
+
+### Group authored and deployed files in the file tree {: #group-authored-and-deployed }
+
+{% Aside %}
+**Note**: This is a {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/XfSWf04g2cwpnFcmp40m.svg", alt="Experimental.", width="20", height="20" %} preview feature available from Chrome version 104.
+{% endAside %}
+
+When developing web applications using frameworks (for example, [React](https://reactjs.org/) or [Angular](https://angular.io/)), it can be difficult to navigate sources due to the minified files generated by the build tools (for example, [Webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/)).
```
Comment (rachelandrew) [inline site/en/docs/devtools/javascript/reference/index.md:316]:
```suggestion
When developing web applications using frameworks (for example, [React](https://reactjs.org/) or [Angular](https://angular.io/)), it can be difficult to navigate sources due to the minified files generated by the build tools (for example, [webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/)).
```

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/console/reference/index.md [hunk 2]
```diff
@@ -87,6 +87,38 @@ messages][15] for a hands-on walkthrough.
...
+
+The **Console** automatically captures stack traces for errors and warnings. A stack trace is a history of function calls (frames) that led to the error or warning. The **Console** shows them in reverse order: the latest frame is at the top.
+
+To view a stack trace, click the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/MaSHbLXzcTFbxhx9K8hX.svg", alt="Expand.", width="24", height="24" %} expand icon next to an error or warning.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/TINzuAQUw7Ug3g2pOljm.png", alt="Stack traces.", width="800", height="401" %}
+
+#### View async stack traces {: #async-stack-traces }
+
+If supported by the framework you are using, DevTools can trace async operations by linking both parts of the async code together.
```
Comment (victorporof) [inline site/en/docs/devtools/console/reference/index.md:100]: "If supported by the framework you are using, or when using browser scheduling primitives directly (such as `setTimeout`), DevTools can trace async operations by linking both parts of the async code together."

File: site/en/docs/devtools/console/reference/index.md [hunk 3]
```diff
@@ -87,6 +87,38 @@ messages][15] for a hands-on walkthrough.
...
+
+In this case, the stack trace shows the "full story" of the async operation.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hyeBPwNtQQ933gjh7Sp7.png", alt="Async stack trace.", width="800", height="861" %}
+
+{% Aside 'gotchas' %}
+DevTools implements this "Async Stack Tagging" feature based on the `console.createTask()` API method. It is up to frameworks to implement the API. For example, [Angular supports this feature](/blog/devtools-better-angular-debugging/#the-async-stack-tagging-api-in-angular).
```
Comment (victorporof) [inline site/en/docs/devtools/console/reference/index.md:107]: I'd reword "It is up to frameworks to make use of this API" to "Frameworks or abstractions are encouraged to make use of this API".

File: site/en/docs/devtools/console/reference/index.md [hunk 4]
```diff
@@ -87,6 +87,38 @@ messages][15] for a hands-on walkthrough.
...
+{% endAside %}
+
+#### Show known third-party frames in stack traces {: #show-third-party }
+
+By default, the **Console** hides from stack traces the third-party frames from sources generated by bundlers (for example, webpack) or frameworks (for example, Angular).
```
Comment (victorporof) [inline site/en/docs/devtools/console/reference/index.md:112]: This requires that the source map generators include the `x_google_ignoreList` field, otherwise this wouldn't work out of the box.

I'd reword this to "When the source maps include the `x_google_ignoreList` field, the **Console** hides from stack traces the third-party frames from sources generated by bundlers (for example, webpack) or frameworks (for example, Angular) by default."

File: site/en/docs/devtools/javascript/reference/index.md [hunk 5]
```diff
@@ -260,6 +258,36 @@ Try restarting the frames of both functions in the following way:
...
+### View async frames {: #async-frames }
+
+If supported by the framework you are using, DevTools can trace async operations by linking both parts of the async code together.
+
+In this case, the **Call Stack** shows the entire call history including async call frames.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/cytE0zCViowAP6ACyW34.png", alt="Async call frames.", width="800", height="615" %}
+
+{% Aside 'gotchas' %}
+DevTools implements this "Async Stack Tagging" feature based on the `console.createTask()` API method. It is up to frameworks to implement the API.
```
Comment (victorporof) [inline site/en/docs/devtools/javascript/reference/index.md:286]: Same comment here about _implementing_ the API.

**Review by victorporof — CHANGES_REQUESTED** [review]
Looks good, a few nits. Thanks!
